server/sonar-server/src/main/java/org/sonar/server/db/migrations/DatabaseMigrator.java

```diff
@@ -46,10 +46,10 @@
   private final ServerUpgradeStatus serverUpgradeStatus;
 
   /**
-   * ServerPluginRepository is used to ensure H2 schema creation is done only after copy of bundle plugins have been done
+   * ServerPluginInstaller is used to ensure H2 schema creation is done only after copy of bundle plugins have been done
    */
   public DatabaseMigrator(DbClient dbClient, MigrationStep[] migrations, ServerUpgradeStatus serverUpgradeStatus,
-    ServerPluginRepository serverPluginRepository) {
+                          ServerPluginRepository unused) {
```

Comment:
why changing the name of the variable (therefor confirming an issue) instead of removing the parameter?